source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -335,9 +336,20 @@ class WinConsoleUIA(KeyboardHandlerBasedTypedCharSupport):
 	#: a lot of text.
 	STABILIZE_DELAY = 0.03
 
+	def _get_apiLevel(self) -> WinConsoleAPILevel:
+		"""
+		This property shows which of several console UIA workarounds are
+		needed in a given conhost instance.
+		See the comments on the WinConsoleAPILevel enum for details.
+		"""
+		return _getConhostAPILevel(self.windowHandle)
```

Comment:
I think this can be a fetch once property. In the current situation, this will be cached for only one core cycle.

```suggestion
		self.apiLevel = _getConhostAPILevel(self.windowHandle)
		return self.apiLevel
```